detection_rules/etc/version.lock.json

```diff
@@ -1176,25 +1305,40 @@
         "rule_name": "Adding Hidden File Attribute via Attrib",
         "sha256": "0c8c7cbbc5634f75e64baccadab65dea2d7b617c6529b847c00105cadd6b1770",
         "type": "eql",
-        "version": 12
+        "version": 14
```

Comment:
forked bump overtaking future ver

Reply:
An opportunity to test: when a rule forks (min_stack version added to already locked rule), we can add `forked_version` (or `max_allowed`), which is saved permanently. We can then use this to ensure previous never exceed it in tests.

Reply:
Looking at this with Mika, it appears this is being bumped with no changes to the rule. The SHA256 hash is the same, but a bump is given. We additionally viewed the history of the rule and it has not received any changed from 8.2 to this release.